packages/ra-core/src/auth/useAuthenticated.ts

```diff
@@ -28,9 +26,21 @@ const emptyParams = {};
  *         </Admin>
  *     );
  */
-export default (params: any = emptyParams) => {
+export const useAuthenticated = <ParamsType = any>(
```

Comment:
this is a breaking change for users who used to pass params, and should be mentioned in the upgrade guide